binplace.targets

```diff
@@ -31,32 +31,24 @@
   </Target>
 
   <Target Name="GetTargetingPackDir">
-    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
-      <TargetingPackDir Include="$(BinDir)netcoreapp\TargetingPack" />
+    <ItemGroup>
+      <TargetingPackDir Include="$(BinDir)ref\$(BuildConfiguration)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
-    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETStandard'))">
-      <TargetingPackDir  Include="$(BinDir)netstandard\TargetingPack" />
-    </ItemGroup>
-    <Error Condition="'@(TargetingPackDir)' == ''" Text="No targeting pack directory could be determined for NuGetTargetMoniker:$(NuGetTargetMoniker)" />
     <Message Text="Got TargetingPackDir:@(TargetingPackDir) from NuGetTargetMoniker:$(NuGetTargetMoniker)" Importance="Low" />
   </Target>
 
   <Target Name="GetRuntimeDir">
-    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
-      <RuntimeDir Include="$(BinDir)netcoreapp\Runtime" />
+    <ItemGroup>
+      <RuntimeDir Include="$(BinDir)runtime\$(BuildConfiguration)" />
```

Comment:
Don't worry about this we have plenty of clean-up left here and I've done part of it with https://github.com/dotnet/corefx/pull/14550.